code/modules/mining/machine_vending.dm

```diff
@@ -24,6 +24,7 @@
 		new /datum/data/mining_equipment("Stabilizing Serum", /obj/item/hivelordstabilizer, 400),
 		new /datum/data/mining_equipment("Fulton Beacon", /obj/item/fulton_core, 400),
 		new /datum/data/mining_equipment("Shelter Capsule", /obj/item/survivalcapsule, 400),
+		new /datum/data/mining_equipment("Environment Proof Bodybag", /obj/item/bodybag/environmental, 500),
```

Comment:
You need to make this a SKYRAT EDIT, otherwise it's all good to merge.

Reply:
done